File: Java/LAB3_arreglos/Lab3/JavaApplication54/src/Lab3/Usuario.java


package Lab3;


public class Usuario {
    //Atributos
    private String nombre;
    private long id;
    private Fecha fecha_nacimiento;
    private String ciudad_nacimiento;
    private long tel;
    private String email;
    private Direccion dir;
    
    //Constructores
    public Usuario(){
    }
    
    public Usuario(String nombre, long id, Fecha fecha_nacimiento, String ciudad_nacimiento, long tel, String email, Direccion dir){
        this.nombre = nombre;
        this.id = id;
        this.fecha_nacimiento = fecha_nacimiento;
        this.ciudad_nacimiento = ciudad_nacimiento;
        this.tel = tel;
        this.email = email;
        this.dir = dir;
        
    }
    
    //Getters and Setters
    public String getNombre(){
        return this.nombre;
    }
    
    public void setNombre(String nombre){
        this.nombre = nombre;
    }
    
    public long getId(){
        return this.id;
    }
    
    public void setId(long id){
        this.id = id;
    }
    
    public Fecha getFecha_nacimiento(){
        return this.fecha_nacimiento;
    }
    
    public void setFecha_nacimiento(Fecha fecha_nacimiento){
        this.fecha_nacimiento = fecha_nacimiento;
    }

    public String getCiudad_nacimiento() {
        return ciudad_nacimiento;
    }

    public void setCiudad_nacimiento(String ciudad_nacimiento) {
        this.ciudad_nacimiento = ciudad_nacimiento;
    }
    
    
    public long getTel(){
        return this.tel;
    }
    
    public void setTel(long tel){
        this.tel = tel;
    }
    
    public String getEmail(){
        return this.email;
    }
    
    public void setEmail(String email){
        this.email = email;
    }
    
    public Direccion getDireccion(){
        return this.dir;
    }
    
    public void setDireccion(Direccion dir){
        this.dir = dir;
    }

    @Override
    public String toString() {
        return "Usuario: " + nombre + ", id :" + id + ", fecha de nacimiento: " + fecha_nacimiento + ", ciudad de nacimiento: " + ciudad_nacimiento + ", tel: " + tel + ", email: " + email + ", direccion: " + dir ;
    }
   

}

File: Python/LAB9_tablas_hash/Usuario.py
class Usuario:
    def __init__(self,nombre,id,fecha_nacimiento,ciudad_nacimiento,tel,email,dir):
        self.__nombre = nombre
        self.__id = id
        self.__fecha_nacimiento = fecha_nacimiento
        self.__ciudad_nacimiento = ciudad_nacimiento
        self.__tel  = tel
        self.__email = email
        self.__dir = dir

    #Get
    def getNombre(self):
        return self.__nombre

    def getId(self):
        return self.__id
    
    def getFecha_nacimiento(self):
        return self.__fecha_nacimiento
    
    def getCiudad_nacimiento(self):
        return self.__ciudad_nacimiento
    
    def getTel(self):
        return self.__tel
    
    def getEmail(self):
        return self.__email
    
    def getDir(self):
        return self.__dir
    
    #Set

    def setNombre(self,n):
        self.__nombre = n
    
    def setId(self,id):
        self.__id = id
    
    def setFecha_nacimiento(self,f):
        self.__fecha_nacimiento = f 
    
    def setCiudad_nacimiento(self,c):
        self.__ciudad_nacimiento = c
    
    def setTel(self,t):
        self.__tel = t 
    
    def setEmail(self,e):
        self.__email = e 
    
    def setDir(self,d):
        self.__dir = d
    
    def __repr__(self):
        return f'''Usuario: {self.__nombre} \n
        ID: {self.__id} \n
        F.Nacimiento: {self.__fecha_nacimiento} \n
        C.Nacimiento: {self.__ciudad_nacimiento} \n
        Tel: {self.__tel} \n
        Email: {self.__email} \n
        Dirección: {self.__dir} \n'''